### PR TITLE
Week2 - Warmup exercise

### PR DESCRIPTION
**Pull request by Saikiruthiga**
created routes for searching an item in the document

### CODE REVIEW

**Comment by Saikiruthiga** [comment]
I mistakenly added week1 warup homework(avg calculation) as well, but i received your feedback and have implemented it. So kindly review only the week2 homework. Sorry for the confusion.

File: nodejs/week2/routers/searchId.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import express from "express";
+const router = express.Router();
+//import documents from "../documents.json" assert { type: "json" };
```
Comment (babak-f) [inline nodejs/week2/routers/searchId.js]: Let's get into the habit of not leaving commented code in the commit

**Review by babak-f — APPROVED** [review]
Great job. Left a minor comment ;)

File: nodejs/week2/routers/searchId.js (hunk 1 above)
Comment (Saikiruthiga) [reply nodejs/week2/routers/searchId.js]: Sure! Thanks for your review

**Pull request merged**